src/_pytest/reports.py

```diff
@@ -368,17 +418,27 @@ def from_item_and_call(cls, item: Item, call: CallInfo[None]) -> TestReport:
                 if excinfo.value._use_item_location:
                     path, line = item.reportinfo()[:2]
                     assert line is not None
-                    longrepr = os.fspath(path), line + 1, r.message
+                    longrepr = (os.fspath(path), line + 1, r.message)
                 else:
                     longrepr = (str(r.path), r.lineno, r.message)
+            elif isinstance(excinfo.value, BaseExceptionGroup):
+                value: BaseExceptionGroup = excinfo.value
```

Comment:
The annotation shouldn't be needed.

More generally I suggest dropping the `value` helper variable, using `excinfo.value` would be more direct.